doc/specs/stdlib_linalg.md

```diff
@@ -206,3 +206,271 @@ program demo_outer_product
     !A = reshape([3., 6., 9., 4., 8., 12.], [3,2])
 end program demo_outer_product
 ```
+
+## `is_square` - Checks if a matrix is square
+
+### Status
+
+Experimental
+
+### Description
+
+Checks if a matrix is square
+
+### Syntax
+
+`d = [[stdlib_linalg(module):is_square(interface)]](A)`
+
+### Arguments
+
+`A`: Shall be a rank-2 array
+
+### Return value
+
+Returns a logical value that is true if the input matrix is square, and false otherwise.
```

Comment:
```suggestion
Returns a `logical` scalar that is `.true.` if the input matrix is square, and `.false.` otherwise.
```

Reply:
Thanks for the suggestion, these changes have been made.